lib/itamae/node.rb

```diff
@@ -1,4 +1,5 @@
 require 'itamae'
+require 'itamae/mash'
```

Comment:
I prefer add `require 'itamae/mash'` to [lib/itamae.rb](https://github.com/itamae-kitchen/itamae/blob/v1.9.11/lib/itamae.rb) than here.

@unasuke Do you think?

Reply:
I agree!